api/authorizers/build.js

```diff
@@ -1,25 +1,27 @@
-const { User, Site } = require("../models")
+const { User, Site } = require('../models');
 
-const authorize = (user, build) => {
-  return User.findById(user.id, { include: [Site] }).then(user => {
-    for (site of user.Sites) {
-      if (build.site || build.Site.id === site.id) {
```

Comment:
This was the problem line. It was resolving the promise just if `build.site` was truthy, but not actually checking to see if the user was associated with the given site.